### DISCUSSION TITLE
是否考虑开启external-controller

### DISCUSSION THREAD

**Discussion by seankuo1** [kkkgo/PaoPaoGateWay#115] [想法]
开启外部控制，便于获取信息
当前是打算在homeassistant中做集成来显示信息
https://github.com/clansty/homeassistant-integration-mihomo

**Comment by kkkgo** [comment]
你看到的web网页就是external-controller 。鉴权就是把密码sha256，或者可以直接抓authorization请求头。

**Reply by seankuo1** [reply]
不知道是不是我写的不对http://192.168.124.101/?token={sha256}/proxies  这样是获取不到数据的{"message":"Unauthorized"}，还是未认证  http://192.168.124.101/ui?token={sha256}/ 访问ui倒是不会报错

**Reply by kkkgo** [reply]
curl 'http://192.168.124.101/configs'  -H 'authorization: Bearer 4704d652efa136a59abde25b2c67164a881a1f5687fc735336efca50f7cc7c28' 

https://wiki.metacubex.one/api/

**Reply by kkkgo** [reply]
没有token=的参数，不知道你哪里抄来的。去看官方文档。浏览器按F12自己看看。

**Reply by seankuo1** [reply]
Thanks♪(･ω･)ﾉ，不是很懂，我以为改写下URL就行，我让AI修改下插件作者的代码可以获取了
